sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/SyntaxHelper.cs

```diff
@@ -202,5 +202,18 @@ public static int GetLineNumber(this SyntaxToken token)
         {
             return token.GetLocation().GetLineSpan().StartLinePosition.Line;
         }
+
+        public static bool IsCatchException(this CatchClauseSyntax catchClause)

```

Comment:
I think I'm missing something - why split? Wouldn't renaming to IsCatchingAllExceptions (I like the name!) be sufficient?